common/src/main/cpp/extensions/openxr_fb_body_tracking_extension_wrapper.cpp

```diff
@@ -0,0 +1,338 @@
+	{ XRBodyTracker::JOINT_RIGHT_MIDDLE_FINGER_METACARPAL, XR_BODY_JOINT_RIGHT_HAND_MIDDLE_METACARPAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_MIDDLE_FINGER_PHALANX_PROXIMAL, XR_BODY_JOINT_RIGHT_HAND_MIDDLE_PROXIMAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_MIDDLE_FINGER_PHALANX_INTERMEDIATE, XR_BODY_JOINT_RIGHT_HAND_MIDDLE_INTERMEDIATE_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_MIDDLE_FINGER_PHALANX_DISTAL, XR_BODY_JOINT_RIGHT_HAND_MIDDLE_DISTAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_MIDDLE_FINGER_TIP, XR_BODY_JOINT_RIGHT_HAND_MIDDLE_TIP_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_RING_FINGER_METACARPAL, XR_BODY_JOINT_RIGHT_HAND_RING_METACARPAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_RING_FINGER_PHALANX_PROXIMAL, XR_BODY_JOINT_RIGHT_HAND_RING_PROXIMAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_RING_FINGER_PHALANX_INTERMEDIATE, XR_BODY_JOINT_RIGHT_HAND_RING_INTERMEDIATE_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_RING_FINGER_PHALANX_DISTAL, XR_BODY_JOINT_RIGHT_HAND_RING_DISTAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_RING_FINGER_TIP, XR_BODY_JOINT_RIGHT_HAND_RING_TIP_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_PINKY_FINGER_METACARPAL, XR_BODY_JOINT_RIGHT_HAND_LITTLE_METACARPAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_PINKY_FINGER_PHALANX_PROXIMAL, XR_BODY_JOINT_RIGHT_HAND_LITTLE_PROXIMAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_PINKY_FINGER_PHALANX_INTERMEDIATE, XR_BODY_JOINT_RIGHT_HAND_LITTLE_INTERMEDIATE_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_PINKY_FINGER_PHALANX_DISTAL, XR_BODY_JOINT_RIGHT_HAND_LITTLE_DISTAL_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+	{ XRBodyTracker::JOINT_RIGHT_PINKY_FINGER_TIP, XR_BODY_JOINT_RIGHT_HAND_LITTLE_TIP_FB, Quaternion(0.5, 0.5, -0.5, 0.5) },
+};
+
+OpenXRFbBodyTrackingExtensionWrapper *OpenXRFbBodyTrackingExtensionWrapper::singleton = nullptr;
+
+OpenXRFbBodyTrackingExtensionWrapper *OpenXRFbBodyTrackingExtensionWrapper::get_singleton() {
+	if (singleton == nullptr) {
+		singleton = memnew(OpenXRFbBodyTrackingExtensionWrapper());
+	}
+	return singleton;
+}
+
+OpenXRFbBodyTrackingExtensionWrapper::OpenXRFbBodyTrackingExtensionWrapper() :
+		OpenXRExtensionWrapperExtension() {
+	ERR_FAIL_COND_MSG(singleton != nullptr, "An OpenXRFbBodyTrackingExtensionWrapper singleton already exists.");
+
+	request_extensions[XR_FB_BODY_TRACKING_EXTENSION_NAME] = &fb_body_tracking_ext;
+
+	singleton = this;
+}
+
+OpenXRFbBodyTrackingExtensionWrapper::~OpenXRFbBodyTrackingExtensionWrapper() {
+	cleanup();
+}
+
+void OpenXRFbBodyTrackingExtensionWrapper::_bind_methods() {
+	ClassDB::bind_method(D_METHOD("is_enabled"), &OpenXRFbBodyTrackingExtensionWrapper::is_enabled);
```

Comment:
This can be removed if the singleton is no longer exposed.

Reply:
The _bind_methods() function cannot be removed as the GDCLASS macro references it; however I have removed the is_enabled registration.